lib/rules/no-get.js

```diff
@@ -80,6 +80,74 @@ function fixGet({
   return fixer.replaceText(node, `${objectTextSafe}${objectPathSeparator}${replacementPath}`);
 }
 
+function reportGetProperties({ context, node, objectText, properties }) {
+  let _properties = properties;
+  if (properties[0].type === 'ArrayExpression') {
+    // When properties are in an array(e.g. getProperties(this.obj, [bar, foo]) ), actual properties are under Array.elements.
+    _properties = properties[0].elements;
+  }
+  const propertyNames = _properties.map((arg) => arg.value);
+
+  context.report({
+    node,
+    message: ERROR_MESSAGE_GET_PROPERTIES,
+    fix(fixer) {
+      return fixGetProperties({
+        fixer,
+        node,
+        objectText,
+        propertyNames,
+      });
+    },
+  });
+}
+
+function fixGetProperties({ fixer, node, objectText, propertyNames }) {
+  if (!propertyNames.every((name) => isValidJSVariableName(name) || isValidJSArrayIndex(name))) {
```

Comment:
I don't think we want `isValidJSArrayIndex` here. It doesn't make sense to have `getProperties(foo, 1)` and we wouldn't want to autofix that.

Reply:
Removed this, and added test case for `getProperties(foo, '1')`